docs/development_suite/deploy/configure-github-actions.md

```diff
@@ -0,0 +1,126 @@
+of the configuration should contain the following information:
+
+```json
+{
+  "deploy": {
+    "type": "github-actions",
+    "providerId": "your-github-provider-id"
+  }
+} 
+```
+
+:::info
+if you haven't set up a GitHub provider yet, [see this page](/development_suite/set-up-infrastructure/configure-provider) to learn how to configure it.
+Remember that the Access Token configured to your provider needs to have the `admin` scope of the organization.
+In order for GitHub to send webhook payloads, your server needs to be accessible from the Internet.
+:::
+
+:::info
+if you haven't set up an environment yet, [see this page](/development_suite/set-up-infrastructure/add-environment) to learn how to configure it.
+:::
+
+## Workflow
+
+In your GitHub repository, create a new file `<root>/.github/workflows/deploy.yml`.
+
+Here is an example that you can use to start:
+```yaml
+name: Deploy
+
+on: deployment
+
+jobs:
+  deploy-ENVIRONMENT_NAME:
+    name: Deploy to ENVIRONMENT_NAME environment
+    if: ${{ github.event.deployment.environment }} == ENVIRONMENT_NAME
+    concurrency: ENVIRONMENT_NAME
+    permissions:
+      deployments: write
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Update deployment status (pending)
+        if: success()
+        uses: chrnorm/deployment-status@v2
+        with:
+          token: '${{ github.token }}'
+          state: 'pending'
+          deployment-id: ${{ github.event.deployment.id }}
+
+      - name: Deploy my app
+        run: |
+          # add your deployment code here
+          echo Deploying...
+          sleep 10s
+          echo Done!
+
+      - name: Update deployment status (success)
+        if: success()
+        uses: chrnorm/deployment-status@v2
+        with:
+          token: '${{ github.token }}'
+          state: 'success'
+          deployment-id: ${{ github.event.deployment.id }}
+
+      - name: Update deployment status (failure)
+        if: failure()
+        uses: chrnorm/deployment-status@v2
+        with:
+          token: '${{ github.token }}'
+          state: 'failure'
+          deployment-id: ${{ github.event.deployment.id }}
+```
+
+When users deploy for the first time, the Console release automatically creates the environments configured inside the project, 
+otherwise you can [follow this tutorial](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#creating-an-environment) 
+to set up your environments.
+
+With the above configuration, when a user presses the `Deploy` button inside the console, the following steps will be executed:
+* The Console creates a new deployments depending on environment chosen on GitHub repository
+* On deployments creation GitHub dispatches a new deployment event and triggers the job specified inside the workflow
+* When the pipeline status of deployments changes, the Console updates to show you its current status
```

Comment:
```suggestion
When users deploy for the first time, the Console release automatically creates the environments configured inside the project, 
otherwise, you can [follow this tutorial](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#creating-an-environment) 
to set up your environments.

With the above configuration, when a user presses the `Deploy` button inside the console, the following steps will be executed:
* The Console creates new deployments depending on the environment chosen on the GitHub repository
* On deployments creation GitHub dispatches a new deployment event and triggers the job specified inside the workflow
* When the pipeline status of deployments changes, the Console updates to show you its current status
```